Build/create-branch.sh

```diff
@@ -42,12 +42,19 @@ push_branch ${BRANCH} "Packages/Framework"
 cd Packages/Application/Neos.Welcome && git checkout -b ${BRANCH} origin/master ; cd -
 push_branch ${BRANCH} "Packages/Application/Neos.Welcome"
 
+# branch behat package
+cd Packages/Application/Neos.Behat && git checkout -b ${BRANCH} origin/master ; cd -
+# special procedure for updating the composer.lock of Behat setup - see https://github.com/neos/behat/issues/23
+cd Packages/Application/Neos.Behat/Resources/Private/Build/Behat && composer update --no-install && git add composer.lock && git commit -m "TASK: Update composer.lock" ; cd -
+push_branch ${BRANCH} "Packages/Application/Neos.Behat"
+
 $(dirname ${BASH_SOURCE[0]})/set-dependencies.sh "${BRANCH}.x-dev" ${BRANCH} "${BUILD_URL}" || exit 1
 
 push_branch ${BRANCH} "Distribution"
 push_branch ${BRANCH} "Build/BuildEssentials"
 push_branch ${BRANCH} "Packages/Framework"
 push_branch ${BRANCH} "Packages/Application/Neos.Welcome"
+push_branch ${BRANCH} "Packages/Application/Neos.Behat"
```

Comment:
Not really sure why there are two `push_branch` commands for every Repo...

Reply:
Are there? What do you mean?

Reply:
e.g. https://github.com/neos/flow-development-distribution/pull/63/files#diff-be9a5ee4e1c54a2f23e06399a5b7c217b2a8f9ccbf282c84b5f8c989b5ce491aR43 and the line just above this - every single block above visits every git directory and checks out the origin/master, then pushes to the new version branch. Then here are all the pushes combined again?

Reply:
Ah, well. That's a better-safe-than-sorry approach. Every "block" of statements forms a task, e.g. "create the branch", "adjust dependencies"–and those are pushed immediately, to avoid forgetting things. That can be adjusted, of course. Then again, a push without changes is almost a no-op.